pkg/apps/apiserver/apiserver.go

```diff
@@ -153,22 +152,3 @@ func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
 	v1Storage["deploymentConfigs/instantiate"] = dcInstantiateStorage
 	return v1Storage, nil
 }
-
-// LegacyLegacyDCRollbackMutator allows us to inject a one-off endpoint into oapi
-type LegacyLegacyDCRollbackMutator struct {
```

Comment:
👍

Reply:
we probably need to test if the `oc rollback` works still with this change (the concern is that `oc` client still use legacy api)

Reply:
> we probably need to test if the oc rollback works still with this change (the concern is that oc client still use legacy api)

test-cmd covers this.